### DISCUSSION TITLE
Sending a donation

### DISCUSSION THREAD

**Discussion by ghost** [OpenTracksApp/OpenTracks#1246] [General]
Could you confirm your bitcoin wallet address? Just want to make sure it goes to the place. :)

**Comment by dennisguse** [comment]
It does; however, I have not yet retrieved any of the donations from there.
Probably should try this out - it is an offline wallet and my first one.

**Comment by ghost** [comment]
The QR code is unreadable to my app, the logo might be too big over it.
https://github.com/OpenTracksApp/OpenTracks/raw/main/promotion/donations-bitcoin.png

**Reply by dennisguse** [reply]
https://raw.githubusercontent.com/OpenTracksApp/OpenTracks/main/README.md

Address is also in the code

**Comment by ghost** [comment]
I use the app on every journey on my bike, I very much appreciate the work put into it. I sent about 0.5 mBTC or $15~.

**Reply by dennisguse** [reply]
Thx; I saw it and will go on the endeavor to convert it something that allows us to buy a coffee ;)

**Comment by dennisguse** [comment]
Quick question: would you also consider donating using the other channels or is bitcoin your only means for donating?
I am not a fan of crypto stuff itself (mainly resource consumption situation) and was more or less on the brink of removing it for good.

**Reply by ghost** [reply]
I'm open to other methods which aren't linked to my bank, card, address, name, etc... Really, only anonymous transfers like Monero or so-so private methods like Bitcoin.
My thinking is using your bank to transact means spending more resources than using crypto does. Heating, lighting, and having staff driving to and from offices, for example. Whereas with crypto, the electricity is only spent on the computers ran by volunteers.

I would suggest accepting Monero as it's truly anonymous form of donating, and possibly re-create the Bitcoin QR code as my app found it unreadable which others may find too.
